src/java_bytecode/java_class_loader.h

```diff
@@ -51,22 +69,22 @@ class java_class_loadert:public messaget
       std::string class_file_name;
     };
 
-    // class name to index map
+    /// Maps class names to a jar file entry
```

Comment:
Thanks for the suggestion.

This comment is 2 lines below the definition of the "jar file entry", the reason why I didn't detail what an entry is. Instead I wrote 4 lines describing to which the jar file entry and this method belong, explaining the intent of the class for someone who (like me) reads this code for the first time. I will add a reference to the class.